testgrid/config/README.md

```diff
@@ -183,6 +183,19 @@ test_groups:
   num_columns_recent: 3
 ```
 
+### See 'Summary' first
+If you'd like to see the 'Summary' tab first when opening a dashboard, instead
+of the first tab of results:
```

Comment:
If we're defaulting `show_summary_first` to true (which SGTM), maybe we want to invert this part of the README to "if you'd like to see the first tab of results instead of the summary tab" etc.